ci-operator/config/openshift/backplane-cli/openshift-backplane-cli-main.yaml

```
@@ -7,7 +7,7 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.18
+    tag: golang-1.21
```

ubi-minimal/ubi base images don't have support for go1.21 yet, only till go1.20.
```suggestion
    tag: golang-1.20
```